README.md

```diff
@@ -14,6 +14,10 @@ writes an `htpasswd` file and turns on HTTP basic auth.
 upstream proxy. DO NOT ENABLE THIS UNLESS BEHIND A REVERSE PROXY THAT PROPERLY
 SETS THIS HEADER.
 
+`NGINX_HTTP_PORT` Change the default port Nginx listens on.  Useful if you are
```

Comment:
How does this work when https is enabled?

Reply:
It looks like the entrypoint script just ignores the `default-http.conf` file when it sees the environment variable to turn on https.